recipes-domd/domd-agl-cluster-demo-platform/domd-agl-cluster-demo-platform.bbappend

```diff
@@ -121,6 +126,10 @@ configure_versions_rcar() {
 
     if [ ! -z "${AOS_VIS_PLUGINS}" ];then
         base_update_conf_value ${local_conf} AOS_VIS_PLUGINS "${AOS_VIS_PLUGINS}"
+        base_add_conf_value ${local_conf} GOVERSION "1.14"
+    fi
+    if [ ! -z "${AOS_VIS_PACKAGE_DIR}" ];then
```

Comment:
Not necessary but if you add an empty line between these two conditions, it would increase readability a little.

Reply:
Fixed. Thanks.